lib/python/Tools/MultiBoot.py

```diff
@@ -203,8 +202,8 @@ def saveKernel(bootSlots, slotCode, kernel):
 							if "rootsubdir" in line:
 								bootSlots[slotCode]["kernel"] = self.getParam(line, "kernel")
 								bootSlots[slotCode]["rootsubdir"] = self.getParam(line, "rootsubdir")
-							elif match("sd[abcd]", line):
-								device = match("sd[abcd]", line).group(0)
+							elif [x for x in ("sda", "sdb", "sdc", "sdd") if x in line]:
+								device = [x for x in ("sda", "sdb", "sdc", "sdd") if x in line][0]
```

Comment:
This can be optimized.

bootDevice = [x for x in ("sda", "sdb", "sdc", "sdd") if x in line]
...
elif bootDevice:
    device = bootDevice[0]

Reply:
I'm not sure but can we get rid of the restriction for sf4008/sf8008 etc and for boxes with 
machine_features emmc